MarkdownApp/Server/Program.cs

```diff
@@ -0,0 +1,49 @@
+using MarkdownApp.Server.Data;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.ResponseCompression;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddControllers();
+builder.Services.AddDbContext<MarkdownContext>(options => options.UseSqlite("Name=MarkDownDB"));
+builder.Services.AddControllersWithViews();
+builder.Services.AddRazorPages();
```

Comment:
do you need this?

Reply:
12 and 14 no, but line 13 yes, i'm using it to map the path for the sqlite database.